client/src/test/resources/suites/AppiumMobileSuiteIOS.xml

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
+<suite thread-count="7" verbose="1" name="AppiumIOSDriverTest Suite" skipfailedinvocationcounts="false" junit="false"
```

Comment:
@alizelzele this suite fails for me in `TestStatePage` and `TestTablePage` ..  I updated the `appPath` and downgraded to `iphone:8.1` .. Ideas?